Config/BuildSettings.swift

```diff
@@ -239,6 +239,8 @@ final class BuildSettings: NSObject {
     static let messageDetailsAllowSave: Bool = true
     static let messageDetailsAllowViewEncryptionInformation : Bool = false
     static let messageDetailsAllowReportContent : Bool = false
+    static let messagesAllowViewRoomRightsChanges : Bool = true
+    static let messagesMinimumPowerLevelAllowViewRoomRightsChanges : Int = 0
```

Comment:
Instead of the magic value 0, can this be looked up from the appropriate enumeration?